extensions/amp-brid-player/0.1/test/test-amp-brid-player.js

```diff
@@ -37,6 +53,21 @@ describe('amp-brid-player', () => {
       if (opt_responsive) {
         bc.setAttribute('layout', 'responsive');
       }
+
+      // see yt test implementation
+      timer.promise(50).then(() => {
+        const bridTimerIframe = bc.querySelector('iframe');
+
+        //console.log('getBridPlayer', bridTimerIframe.contentWindow);
```

Comment:
Rebased and removed the above code. Also we have pushed a new version with unmute fix.